tests/Unit/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Test_KerrSchild.cpp

```diff
@@ -155,6 +168,72 @@ void test_schwarzschild(const DataType& used_for_size) {
   CHECK_ITERABLE_APPROX(d_g, expected_d_g);
 }
 
+template <typename FrameType>
+void test_numerical_deriv_det_spatial_metric(const DataVector& used_for_size) {
+  // Parameters for KerrSchild solution
+  const double mass = 1.01;
+  const std::array<double, 3> spin{{0.0, 0.0, 0.0}};
+  const std::array<double, 3> center{{0.0, 0.0, 0.0}};
+  gr::Solutions::KerrSchild solution(mass, spin, center);
+
+  // Setup grid
+  const size_t num_points_1d = 8;
+  const std::array<double, 3> lower_bound{{0.82, 1.24, 1.32}};
+  const std::array<double, 3> upper_bound{{0.8, 1.22, 1.30}};
+  const size_t SpatialDim = 3;
+  Mesh<SpatialDim> mesh{num_points_1d, Spectral::Basis::Legendre,
+                        Spectral::Quadrature::GaussLobatto};
+  const auto coord_map =
+      domain::make_coordinate_map<Frame::ElementLogical, FrameType>(Affine3D{
```

Comment:
Just to provide another bit of info on this mystery: In my tests I tried a `Wedge` map and hit a surprisingly large floor when taking numerical derivatives of Christoffel symbols. Perhaps also for your case here it would be instructive to try a `Wedge` map. Another test I was considering was to code up the analytic Ricci scalar for non-spinning KerrSchild and compare to the one computed via numerical derivatives.

Reply:
I think I might be misinterpreting this note about `Wedge`. I interpreted your first sentence to mean that you were seeing large error floors with `Wedge`, but then you're recommending trying it out. If `Wedge` seems to be giving large errors, I imagine then the reason for using it is not to improve the current error. If this is correct, could you explain what the benefit of trying it is? Sorry I'm not connecting the dots!

That Ricci scalar test sounds like a good idea! I've been hitting some large error floors with some numerical derivative computations in the CCZ4 system.

Reply:
Oh I think maybe I follow you. Are you saying the potential benefit of using a `Wedge` map would come in when running a simulation for debugging numerical errors, and getting to see the error floor be reached with less # of points (i.e. we can spend less real time running a simulation for debugging numerical errors)?

Reply:
I see what you mean, now. Thank you for your patience and willingness to explain! The reasoning wasn't clicking for me before. I'll look into this, as well, and let you know what I find.

Reply:
I ended up not looking into this, as @nilsdeppe and I found that the source of my numerical error is the fact that the Legendre polynomials don't fit the derivative of 1 / r well.